problem105.rb

```diff
@@ -0,0 +1,19 @@
+# Create a function that takes a binary string and returns the text. The eight bits on the binary string represent 1
+# character on the ASCII table. For further info, check out the resource tab.
+# Examples
+# binary_text("01101110011011110110010001100101") ➞ "node"
+# binary_text('0111001001100101011000010110001101110100') ➞ "react"
+# binary_text("011100000111100101110100011010000110111101101110") ➞ "python"
+def binary_text(str)
+  a = []
+  a << str
+  a.pack("B*")
```

Comment:
Are you familiar with this method? Because you can use builtin method but I already told you try to make your own logic. Working is fine, if you know the backend logic of this `pack` method so this is good otherwise push code with your own logic.

Reply:
another method with own logic is already written in comment section.Kindly take a look a them